doc/reference/tooling/tt_cli/connect.rst

```diff
@@ -8,8 +8,8 @@ Connecting to a Tarantool instance
     $ tt connect {URI|INSTANCE} [OPTION ...]
 
 
-``tt connect`` connects to a Tarantool instance by its URI or name specified
-during its startup (``tt start``).
+``tt connect`` connects to a Tarantool instance by its URI or instance name specified
+in the current environment.
 
 Options
```

Comment:
Let's expand `-u` and `-p` descriptions a bit. `Username` four times in a row looks a bit weird :)

<img width="719" alt="image" src="https://github.com/tarantool/doc/assets/38073144/687d5afd-b798-461c-ac44-50bb65faa7fa">